formulaire-public/src/main/java/fr/openent/formulaire_public/service/impl/DefaultResponseService.java

```diff
@@ -40,7 +40,8 @@ public void createResponses(JsonArray responses, JsonObject distribution, Handle
                         .add(response.getInteger(CHOICE_ID, null))
                         .add(response.getString(ANSWER, ""))
                         .add("")
-                        .add(distribution.getInteger(ID, null));
+                        .add(distribution.getInteger(ID, null))
+                        .add(response.getInteger(CHOICE_INDEX, null));
```

Comment:
idem

Reply:
Cf plus haut, le champ SQL à une règle NOT NULL